java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_4/Reporter.java

```diff
@@ -0,0 +1,68 @@
+package com.zoolatech.lecture3.tasks._4;
+
+import java.util.Scanner;
+
+public abstract class Reporter {
```

Comment:
As far as I can see, right now all 3 subclasses have the behaviour. But, they must do different things (e.g., print "“Adding file details in CSV format”) which depend on type of documents they work with.

Reply:
didn't understand the task. I hope that it is correct now